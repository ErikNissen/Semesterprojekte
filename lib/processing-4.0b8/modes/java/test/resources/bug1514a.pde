version https://git-lfs.github.com/spec/v1
oid sha256:a7777799646ec026d10cd2c7bb9cca789ac0ad9a5209a7f9582415a96f4971c2
size 219
