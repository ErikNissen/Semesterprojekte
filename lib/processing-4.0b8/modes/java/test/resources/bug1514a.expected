version https://git-lfs.github.com/spec/v1
oid sha256:88c97b3567b4597e17821a83fd2ec804921bf99df2a5af221d952f8327db4d1a
size 981
